README.md

```diff
@@ -308,10 +309,21 @@ Use this shell to run commands like `bundle exec rails c` or `yarn run ember tes
 
 Remember to prefix commands with `bundle exec` or `yarn exec`
 
+can also optionally take arguments, if arguments are passed it will run those as a command in the shell instead of opening an bash shell.
+
 #### Usage:
 
 ```bash
-pndev reset
+# to start a normal shell
+pndev shell
+# or
+pndev sh
+
+# to run a command in a shell
+pndev sh rspec spec/models/activity_spec.rb
+
+# if the command you want to run takes flags use --
+pndev sh -- rake -T
```

Comment:
this works for commands that are not just `sh` right, but other that are defined in toml file?  Like `pndev test -- --quiet`?

Reply:
I don't think so. `pndev test` is not a thing. you'd have to use `pndev run test` which doesn't seem to support appending more flags.

```
 cargo run -- sh -- ls -la
```

Works 

But if I put `test` in pndev.toml

```
test = "ls"
```

```cargo run -- run test```

works

But I cannot do 

```cargo run -- run test -la```

OR 

```cargo run -- run test -- -la```

I think we'd need to add the same support for `run` that @ghedamat just added for `sh`

Reply:
oh, I misunderstood how this works.  In this case the `--` stops the flag processing by pndev making them available to the command.  

Still, it would make sense that `--` anywhere would do that, and pass whatever comes after it to the command being `run`

Reply:
Agreed.

Reply:
I would consider those changes out of scope for this pr as they are for a different command. You guys ok if we punt?

Reply:
For sure